ilp-core/src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -198,7 +196,7 @@ default InterledgerAddress with(final String addressSegment) {
    *     {@code false}.
    */
   default boolean hasPrefix() {
```

Comment:
I'm thinking we should just remove this method (`hasPrefix`) entirely. I don't think it's possible for `InterledgerAddress` to be constructed without a prefix, so asking if it has one doesn't seem to be valuable.

Thoughts?

Reply:
I agree. Made the changes for the same and amended the detailed commit message. 😄